<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecialAddressBehaviorDialog</class>
 <widget class="QDialog" name="SpecialAddressBehaviorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special Address Behavior</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="radioButtonBehaveNormalAddress">
     <property name="text">
      <string>Behave as a normal address</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioButtonBehaviorMailingList">
     <property name="text">
      <string>Behave as a pseudo-mailing-list address</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mail received to a pseudo-mailing-list address will be automatically broadcast to subscribers (and thus will be public).</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name of the pseudo-mailing-list:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEditMailingListName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>368</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonBehaveNormalAddress</tabstop>
  <tabstop>radioButtonBehaviorMailingList</tabstop>
  <tabstop>lineEditMailingListName</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpecialAddressBehaviorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpecialAddressBehaviorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonBehaviorMailingList</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEditMailingListName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonBehaveNormalAddress</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEditMailingListName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
